nixos/modules/system/boot/systemd.nix

```diff
@@ -102,6 +102,10 @@ let
       "systemd-rfkill.service"
       "systemd-rfkill.socket"
 
+      # Boot counting
+      "boot-complete.target"
+    ] ++ lib.optional config.boot.loader.systemd-boot.bootCounting.enable "systemd-bless-boot.service" ++ [
```

Comment:
Don't we also need `systemd-boot-check-no-failures.service` when `bootCounting.enable = true` ?  Or at least have an option to enable it?

Reply:
> Note that due the simple nature of this check this service is probably not suitable for deployment in most scenarios. It is primarily useful only as example for developing more fine-grained checks to order before boot-complete.target.

This tells me we should maybe only add it as an option, yes.

Reply:
`systemd-boot-check-no-failures.service` is probably way too strict and will not be useful in a generic system.